grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -455,14 +457,32 @@ static HttpHeaders statusToTrailers(Status status, boolean headersSent) {
                     .set(HttpHeaderNames.CONTENT_TYPE, "application/grpc+proto");
         }
         trailers.add(GrpcHeaderNames.GRPC_STATUS, Integer.toString(status.getCode().value()));
-        if (status.getDescription() != null) {
-            trailers.add(GrpcHeaderNames.GRPC_MESSAGE,
-                         StatusMessageEscaper.escape(status.getDescription()));
+
+        final String description = statusToDescription(status);
+        if (description != null) {
+            trailers.add(GrpcHeaderNames.GRPC_MESSAGE, StatusMessageEscaper.escape(description));
         }
 
         return trailers;
     }
 
+    @Nullable
+    private static String statusToDescription(Status status) {
+        final String description = status.getDescription();
+        if (Flags.verboseResponses()) {
+            final Throwable cause = status.getCause();
+            if (cause != null) {
+                final String trace = Throwables.getStackTraceAsString(cause);
+                if (Strings.isNullOrEmpty(description)) {
+                    return "Caused by: " + trace;
+                } else {
+                    return description + System.lineSeparator() + "Caused by: " + trace;
```

Comment:
Oops one point - I'd go with `\n` instead of `System`

Reply:
I thought about this and realized `Throwables.getStackTraceAsString()` uses platform dependent line ending, so just used `System.lineSeparator()` for consistency. I think I can add something similar to `com.linecorp.armeria.common.util.Exceptions` to produce a platform-independent output. Sounds good?

Reply:
- Un-deprecated `Exceptions.traceText()` and made it always use `\n` as a line delimiter.
- Replaced all `Throwables.getStackTraceAsString()` and most `System.lineSeparator()` calls